services/distribution/src/main/resources/application.yaml

```diff
@@ -20,6 +20,8 @@ services:
     output-file-name-v2: index-v2
     # The number of days to retain diagnosis keys and trace time warnings for both database persistency layer and files stored on the object store.
     retention-days: 14
+    # The number of days for diagnosis keys (based upon submission timestamp) to publish on CDN.
+    days-to-publish: ${DAYS_TO_PUBLISH:10}
```

Comment:
This effectively means CWA does not take the last 14 days into account but only the last 10, correct?

Reply:
yes - but still under discussion and not clear if it will come or not... but I'd like to be prepared when things get urgent :-)

Reply:
Okay, thanks @hilmarf!

Reply:
Is this change now live? If yes, several texts in the app need updating, as well as the website.